File: Selenium/Activities/activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, timeout=10)

    driver.get("https://training-support.net/webelements/alerts")

    print("Page title is: ", driver.title)

    driver.find_element(By.ID, "confirmation").click()

    confirmAlert = wait.until(EC.alert_is_present())

    alertText = confirmAlert.text
    print("Text in alert: " + alertText)

    confirmAlert.dismiss()

    print(driver.find_element(By.ID, "result").text)

File: Selenium/Activities/activity19.java
package activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class activity19 {
	public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/alerts");
        System.out.println("Page title: " + driver.getTitle());
        
        driver.findElement(By.id("confirmation")).click();
        Alert confirmAlert = driver.switchTo().alert();
        System.out.println(confirmAlert.getText());
        //confirmAlert.accept();	
        confirmAlert.dismiss();
        Thread.sleep(2000);
        
        System.out.println(driver.findElement(By.id("result")).getText());
       
        driver.quit();

}
}
